### PR TITLE
node: Allow to configure tombsone lifetime

### PR DESCRIPTION
**Pull request by fyrchik**
Currently, DELETE service sets tombstone expiration epoch to `current epoch + 5`. This works less than ideal in private networks where an epoch can be e.g. 10 minutes. In this case, after a node is unavailable for more than 1 hour, already deleted objects have a chance to reappear.

After this commit tombstone lifetime can be configured.

Signed-off-by: Evgenii Stratonikov <e.stratonikov@yadro.com>

### CODE REVIEW

File: docs/storage-node-configuration.md [hunk 1]
```diff
@@ -415,14 +415,15 @@ replicator:
...
     pool_size_remote: 100
 ```
 
-| Parameter              | Type  | Default value | Description                                                                                    |
-|------------------------|-------|---------------|------------------------------------------------------------------------------------------------|
-| `put.pool_size_remote` | `int` | `10`          | Max pool size for performing remote `PUT` operations. Used by Policer and Replicator services. |
\ No newline at end of file
+| Parameter                   | Type  | Default value | Description                                                                                    |
+|-----------------------------|-------|---------------|------------------------------------------------------------------------------------------------|
+| `delete.tombstone_lifetime` | `int` | `5`           | Tombstone lifetime for removed objects.                                                        |
```
Comment (anikeev-yadro) [inline docs/storage-node-configuration.md:428]: Please clarify units for this parameter

Comment (fyrchik) [reply docs/storage-node-configuration.md]: Fixed!

**Review by roman-khimov — COMMENTED** [review]
If it's a network-specific parameter, then it should be a network-wide configuration, isn't it? Otherwise you can have different nodes using different configurations on the same network and it can lead to some surprises.

**Comment by fyrchik** [comment]
@roman-khimov I partially agree with you. The case is that this lifetime _can_ be arbitrary, we set it only in the DELETE service for every tombstone. Like with REGULAR objects the expiration epoch is not restricted.

**Comment by roman-khimov** [comment]
Or take it from another perspective, you have some local configuration now that potentially affects the network globally. Is it a good thing to have?

**Comment by fyrchik** [comment]
Ok, I agree, it would probably also make it easier to scale.

This fix is kinda HOT, so I will check how invasive it is to take a network setting into account here.

**Review by realloc — APPROVED** [review]

**Comment by realloc** [comment]
This parameters must be treated as a local node configuration option. While set network-wise it doesn't prevent any node from misbehavior nor affects interoperability.

**Pull request merged**